ext/RMagick/rmilist.c

```diff
@@ -368,6 +369,7 @@ ImageList_fx(int argc, VALUE *argv, VALUE self)
     ChannelType channels;
     ExceptionInfo *exception;
 
+    rb_warning("ImageList#fx is deprecated; use Image#fx");
```

Comment:
I haven't thought too deeply about this, but would it make sense to have `ImageList#fx` loop through the images and apply it to all of the images? Or do we want to make the user do that in their own code?

If we do add the loop, I think this PR still makes sense, since it probably makes sense for the `fx` method to be defined on `Image`.

Reply:
I personally think that if a users wants to do a `fx` operation on all the images they should do the loop. Otherwise we would end up with all the `Image` methods on the `ImageList` class instead of the ones that do something special with a list.

Reply:
:+1: